docs/content/contributing/maintainers-guidelines.md

```diff
@@ -0,0 +1,68 @@
+- You will be listed on our Maintainers Github page as well as on our website in the section [maintainers](maintainers.md).
+- We will be promoting you in social channels (mostly on Twitter).
+
+## Governance
+
+- Roadmap meetings on a regular basis where all maintainers are welcome.
+
+## Communicating
+- All of our maintainers are added to Slack #traefik-maintainers channel that belongs to Traefik labs workspace. Having the team in one place helps us to communicate effectively. You can reach Traefik core developers directly, which offers the possibility to discuss issues, pull requests, enhancements more efficiently and get the feedback almost immediately. Fewer blockers mean more fun and engaging work.
+- We publish a report to the channel on a daily basis that includes all the activities performed during the day. You are updated in regards to the workload that has been processed including: working on the new features and enhancements, activities related to the reported issues and PR’s, other important project-related announcements.
+- At 5:00 PM CET every day we review all the created issues that have been reported, assign the specific *[labels](maintainers.md#Labels)* and prioritize them based on the severity of the problem. The process is called *[issue triaging](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).* Each of the maintainers is welcome to join the meeting. For that purpose, we use a dedicated Discord server where you are invited once you have become the official maintainer.
+
+## Maintainers Activity
+In order to keep the core team efficient and dynamic, maintainers activity and involvement will be reviewed on a regular basis.
+
+- Has the maintainer engaged with the team and the community by meeting two or more of these benchmarks in the past six months?
+    - Has the maintainer participated in at least two or three maintainer meetings?
+    - Substantial review of at least one or two PRs from either contributors or maintainers.
+    - Opened at least one or two bug fixes or feature request PRs that were eventually merged (or on a trajectory for merge).
+    - Substantial participation in the Help Wanted program (answered questions, helped identify issues, applied guidelines from the Help Wanted guide to open issues).
+    - Substantial participation with the community in general.
+- Has the maintainer shown a consistent pattern of helpful, non-threatening, and friendly behavior towards other people on the maintainer team and with our community?
+
+## Additional comments for (not only) maintainers
+
+- Use Traefik on a daily basis. You can’t be a good maintainer if you are not using Traefik daily and you can’t put yourself in your users’ shoes.
+- Prioritize the maintainer. Be user-focused but if you use Traefik every day, you are already in the user's shoes, so respect other maintainers and other community members.
```

Comment:
I'm not sure about the daily basis usage, perhaps we can rephrase with something like.
```suggestion
- Be able to put yourself in users’ shoes.
- Be open-minded and respectful with other maintainers and other community members.
```